Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1232,6 +1234,31 @@ int NetPlayClient::LocalPadToInGamePad(int local_pad) const
   return ingame_pad;
 }
 
+int NetPlayClient::NumLocalWiimotes() const
+{
+  return static_cast<int>(
+      std::count_if(m_wiimote_map.begin(), m_wiimote_map.end(),
+                    [this](auto mapping) { return mapping == m_local_player->pid; }));
+}
+
+int NetPlayClient::InWiimoteToLocalWiimote(int ingame_wiimote) const
+{
+  // not our pad
+  if (m_wiimote_map[ingame_wiimote] != m_local_player->pid)
+    return 4;
```

Comment:
This is rather magical and difficult to comprehend. Also the result is never checked for, anything bad that can happen if more than one ingame_wiimote isn't local?